main.go

```diff
@@ -1,107 +1 @@
-package main
 
-import (
-	"database/sql"
-	"fmt"
-	"log"
-	"os"
-	
-
-	_ "github.com/mattn/go-sqlite3"
-)
-
-func openDB(dbfile string) (*sql.DB) {
-	db, err := sql.Open("sqlite3", dbfile)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	return(db)
-}
-
-func getHandleID(db *sql.DB, handle string) ([]int){
```

Comment:
Very nit, but instead of 
`func getHandleID(db *sql.DB, handle string) ([]int){`
do 
`func getHandleID(db *sql.DB, handle string) []int {`
when you are only returning one thing